File: problemas/1000/1099.php
<?php

$n = readline();

for ($i = 0; $i < $n; $i++) {
    $linha = explode(' ', readline());
    $menor = min($linha[0], $linha[1]);
    $maior = max($linha[0], $linha[1]);
    $soma = 0;

    for ($j = $menor + 1; $j < $maior; $j++) {
        if ($j % 2 != 0) {
            $soma += $j;
        }
    }

    echo $soma . "\n";
}

File: problemas/1000/1099.cs
using System;

namespace uri1099
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());

            for (int i = 0; i < n; i++)
            {
                string[] linha = Console.ReadLine().Split(' ');
                int x = int.Parse(linha[0]);
                int y = int.Parse(linha[1]);
                int menor = Math.Min(x, y);
                int maior = Math.Max(x, y);
                int soma = 0;

                for (int j = menor + 1; j < maior; j++)
                {
                    if (j % 2 != 0)
                    {
                        soma += j;
                    }
                }

                Console.WriteLine(soma);
            }
        }
    }
}